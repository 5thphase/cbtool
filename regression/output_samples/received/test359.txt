The "sim" cloud named "TESTCLOUD" was successfully attached to this experiment.
The experiment identifier is EXP-02-13-2014-02-54-16-PM-EST

