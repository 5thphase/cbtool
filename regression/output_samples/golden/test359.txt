 status: VPN configuration for this cloud already generated: /home/msilva/cloudbench/lib/auxiliary//../../configs/generated/TESTCLOUD_server-cb-openvpn.conf
The "sim" cloud named "TESTCLOUD" was successfully attached to this experiment.
The experiment identifier is EXP-02-04-2016-04-51-28-PM-UTC

